api/openapi-spec/webhook.yaml

```diff
@@ -41,7 +41,6 @@ components:
           type: string
       required:
         - revision
-        - origin_url
```

Comment:
We do not have `origin_url` in message for authenticated datafiles. Relaxing the schema.